pipelines/broad/qc/CheckFingerprint.documentation.md

```diff
@@ -0,0 +1,58 @@
+# CheckFingerprint Pipeline
+
+This document describes version 1.0.0 of the CheckFingerprint Pipeline. The CheckFingerprint Pipeline is used internally within the Broad Institute. It is written in WDL and designed to work on the [Cromwell execution engine](https://cromwell.readthedocs.io/en/stable/) using Google Cloud Services and Google Cloud Storage as its backend and storage.
+
+The CheckFingerprint Pipeline runs the Picard tool 'CheckFingerprint' against a supplied input file (VCF, CRAM, BAM or SAM) using a set of 'fingerprint' genotypes. These genotypes can either be generated by pulling them from the (Broad-internal) Mercury Fingerprint Store or be supplied as inputs to the pipeline.
+
+## Modes
+
+The CheckFingerprint Pipeline can operate in different modes (effectively) based on the inputs supplied. These are briefly described here.
```

Comment:
```suggestion
The CheckFingerprint Pipeline can operate in different modes based on the inputs supplied. These options include:
1. Download a fingerprint file from the Mercury Fingerprint Store **OR** supply one as input (i.e. define the fingerprint_genotype_VCF input). 
2. Compare the genotypes in the selected fingerprint file to those in an input VCF/BAM **OR** just return the fingerprint file (i.e. do not supply a comparison file).

Read more about the different input options in the section below.

Reply:
I added this suggestion because I felt like the actual Modes weren't described in the Modes section. We just kind of said that there are modes but left it at that. I took a stab at what I thought the big picture modes were, but please feel free to revise.

Reply:
Oops, yeah - I meant to fill in the modes section more / was kind of on fence about doing that or describing more fully in the inputs section (which I attempted to do). I like your summary of the modes though.